common/quotas/calculator/logged_calculator.go

```diff
@@ -99,7 +99,7 @@ func (c *LoggedNamespaceCalculator) getOrCreateQuotaLogger(
 
 	quotaLogger, ok := c.quotaLoggers[namespace]
 	if !ok {
-		quotaLogger = newQuotaLogger(c.logger)
+		quotaLogger = newQuotaLogger(log.With(c.logger, tag.WorkflowNamespace(namespace)))
 		c.quotaLoggers[namespace] = quotaLogger
 	}
 
```

Comment:
could you fix the `newQutoa` typo below?

Reply:
Oops. Thanks for catching! Fixed.